src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/StandardTransformFactory.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using CoreWCF.IdentityModel.Tokens;
+using System.Security.Cryptography;
+
+namespace CoreWCF.IdentityModel
+{
+    class StandardTransformFactory : TransformFactory
+    {
```

Comment:
Not used by the Crypto SignedXml implementation.

Reply:
deleted